lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -2741,13 +2744,15 @@ def ValidateParameters(cls, hvparams):
     vnc_bind_address = hvparams[constants.HV_VNC_BIND_ADDRESS]
     if vnc_bind_address:
       bound_to_addr = netutils.IP4Address.IsValid(vnc_bind_address)
```

Comment:
This line is not necessary anymore, is it? Since the check is done in the following statement anyway.